content/en/docs/concepts/traffic-management/index.md

```diff
@@ -13,237 +13,143 @@ aliases:
-proxies route traffic to the subset according to the rule but again use
-round-robin to route traffic across the instances of each subset.
-
-This configuration method provides the following advantages:
-
-- Decouples the application code from the evolution of the application's dependent services.
-- Provides monitoring benefits.
-For details, see [Mixer policies and telemetry](/docs/reference/config/policy-and-telemetry/).
-
-For example, in A/B testing we often want to configure traffic routes based on
-percentages. With Istio, you can use a virtual service to specify a routing
-rule that sends 25% of requests to instances in the `v2` subset, and sends the
-remaining 75% of requests to instances in the `v1` subset. The following
-configuration accomplishes our example for the `reviews` service.
+Istio’s traffic routing rules let you easily control the flow
+of traffic and API calls between services. Istio simplifies configuration of
+service-level properties like circuit breakers, timeouts, and retries, and makes
+it easy to set up important tasks like A/B testing, canary rollouts, and staged
+rollouts with percentage-based traffic splits. It also provides out-of-box
+failure recovery features, which, combined with Istio’s
+[observability](/docs/concepts/observability/) features, helps make your application
+more robust against failures of dependent services or the network.
+
+Istio’s traffic management model relies on the {{< gloss >}}Envoy{{</ gloss >}}
+proxies that are deployed along with your services. All traffic that your mesh
+services send and receive ({{< gloss >}}data plane{{</ gloss >}} traffic) is proxied through Envoy, making
+it easy to direct and control traffic around your mesh without making any
+changes to your services.
+
+If you’re interested in the details of how the features described in this guide
+work, you can find out more about Istio’s traffic management architecture in the
+[Architecture](#architecture) section at the end of this document. The rest of
+this guide introduces Istio’s traffic management features.
+
+## Introducing Istio Traffic Management
+
+In order to direct traffic around your mesh, Istio needs to know where all your
+endpoints are, and which services they belong to. To populate its own
+{{< gloss >}}service registry{{</ gloss >}}, Istio connects to a service
+discovery system. For example, if you've installed Istio on a Kubernetes cluster,
+then by default Istio knows about all the services and endpoints in that cluster.
+
+Using this service registry, the Envoy proxies can then direct traffic to the
+relevant services. Most microservice-based applications have multiple instances
+of each service workload to handle service traffic, sometimes referred to as a
+load balancing pool. By default, the Envoy proxies distribute traffic across
+each service’s load balancing pool using a round-robin model, where requests are
+sent to each pool member in turn, returning to the top of the pool once each
```

Comment:
something here feels technically off, although I'm not sure what it is.  Perhaps this paragraph is too much detail for the concepts section.  Any other reviewers have actionable suggestions for rework here?

Reply:
I think it's useful info because it explains what happens if you just use Istio out of the box and don't do any traffic configuration at all - your data plane traffic still gets sent to workload instances, it's just that you can't do anything more complicated than "send to that service w round robin load balancing between all its instances". We then excite and delight the users in the next paragraph with all the lovely things they can do if they use our traffic management resources. :)

Reply:
@frankbu, thoughts?

Reply:
It's good as is.